### PR TITLE
fix: filter CSS related 499 chunk errors

### PR DESCRIPTION
**Pull request by vm**
## Description
Similar to https://github.com/Uniswap/interface/pull/6365, except handles CSS chunks additionally. Abstracts the function that checks window.performance.

_JIRA ticket:_ https://uniswaplabs.atlassian.net/browse/INFRA-172

### Automated testing
- [x] Unit test
- [x] Integration/E2E test (n/a)

### CODE REVIEW

File: src/tracing/errors.ts [hunk 1]
```diff
@@ -56,18 +69,14 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
...
-       * If there's no status (meaning the browser doesn't support `responseStatus`) then we also ignore.
-       * These errors are likely also 499 errors, and we can catch any spikes in non-499 chunk errors via other browsers.
-       */
-      if (!status || status === 499) {
-        return null
-      }
+    if (error.message.match(/Loading CSS chunk \d+ failed\. \(.+\.chunk\.css\)/)) {
+      const relativePath = error.message.match(/\/static\/css\/.*\.chunk\.css/)?.[0]
+      const asset = `https://app.uniswap.org${relativePath}`
+      console.log(asset)
```
Comment (vm) [inline src/tracing/errors.ts:78]: rm

**Review by just-toby — APPROVED** [review]

File: src/tracing/errors.test.ts [hunk 2]
```diff
@@ -71,6 +82,17 @@ describe('filterKnownErrors', () => {
       expect(filterKnownErrors(ERROR, { originalException })).not.toBeNull()
     })
 
+    it('keeps CSS error when status is different than 499', () => {
+      jest.spyOn(window.performance, 'getEntriesByType').mockReturnValue([
+        {
+          name: 'https://app.uniswap.org/static/css/12.d5b3cfe3.chunk.css',
+          responseStatus: 200,
```
Comment (cmcewen) [inline src/tracing/errors.test.ts:89]: 200 wouldn't be an error 😄. but nbd

File: src/tracing/errors.ts [hunk 3]
```diff
@@ -56,18 +69,13 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
...
-      /*
-       * If the status if 499, then we ignore.
-       * If there's no status (meaning the browser doesn't support `responseStatus`) then we also ignore.
-       * These errors are likely also 499 errors, and we can catch any spikes in non-499 chunk errors via other browsers.
-       */
-      if (!status || status === 499) {
-        return null
-      }
+    if (error.message.match(/Loading CSS chunk \d+ failed\. \(.+\.chunk\.css\)/)) {
+      const relativePath = error.message.match(/\/static\/css\/.*\.chunk\.css/)?.[0]
```
Comment (cmcewen) [inline src/tracing/errors.ts:76]: why do you do relative path here but the full asset path above for JS? also is it easier just to do for any asset? or you want to specifically do this for css/js only?

Comment (vm) [reply src/tracing/errors.ts:76]: the actual error in sentry is different between the two

Comment (vm) [reply src/tracing/errors.ts:76]: https://uniswap-labs.sentry.io/issues/4066689493/?project=4504255148851200&query=is%3Aunresolved&referrer=issue-stream&stream_index=6
https://uniswap-labs.sentry.io/issues/4009148084/?project=4504255148851200&query=is%3Aunresolved&referrer=issue-stream&stream_index=13

Comment (cmcewen) [reply src/tracing/errors.ts:76]: oh that's annoying by the loader

**Review by cmcewen — APPROVED** [review]

**Pull request merged**